store/search.js

```diff
@@ -112,11 +103,47 @@ export const actions = {
     commit('setPill', null);
   },
 
+  async deriveApiSettings({ commit, dispatch, state }) {
+    // Coerce qf from user input into an array as it may be a single string
+    const userParams = Object.assign({}, state.userParams || {});
+    userParams.qf = [].concat(userParams.qf || []);
+
+    const apiParams = merge(userParams, state.overrideParams || {});
+    commit('setApiParams', apiParams);
+    commit('setApiOptions', {});
+
+    if (apiParams.theme === 'newspaper') {
+      await dispatch('deriveApiSettingsForNewspaperTheme');
+    }
+  },
+
+  deriveApiSettingsForNewspaperTheme({ commit, state }) {
+    const apiParams = Object.assign({}, state.apiParams);
+    const apiOptions = Object.assign({}, state.apiOptions);
+
+    // Ensure newspapers collection gets fulltext API by default
+    if (!apiParams.api) {
+      apiParams.api = 'fulltext';
+    }
+
+    if (apiParams.api === 'fulltext') {
+      // TODO: fulltext search API should be aware of contentTier, but is not.
+      //       If & when it is, this can be removed.
+      apiParams.qf = ([].concat(apiParams.qf)).filter(qf => !/^contentTier:/.test(qf));
+      apiParams.qf.push('contentTier:*');
+
+      apiOptions.origin = 'https://newspapers.eanadev.org';
```

Comment:
Should this be moved to the .env in the end? Similar to the "EUROPEANA_API_ORIGIN"?

Reply:
Agreed and done.